app/models/episode_entry_handler.rb

```diff
@@ -63,28 +62,12 @@ def update_link
     episode.url = nil if episode.url&.match(/libsyn\.com/)
   end
 
-  def update_enclosure
-    enclosure_hash = overrides.fetch(:enclosure, {}).dup
-    if overrides[:feedburner_orig_enclosure_link]
-      enclosure_hash[:url] = overrides[:feedburner_orig_enclosure_link]
-    end
-
-    if overrides[:enclosure]
-      episode.enclosure = Enclosure.build_from_enclosure(episode, enclosure_hash)
-    else
-      episode.enclosures.destroy_all
-    end
-  end
-
   def update_image
     episode.image = overrides[:image_url]
   end
 
   def update_contents
     new_contents = Array(overrides[:contents]).sort_by { |c| c[:position] }
-
-    episode.contents = new_contents.map do |c|
-      Content.build_from_content(episode, c)
-    end
+    episode.media = new_contents.map { |c| c[:url] }
```

Comment:
Invoking the `media=` setter to do the heavy lifting - checking if the `content.original_url` has changed, replacing/deleting/creating contents, etc.